addons/digest/wizard/digest_test.py

```diff
@@ -0,0 +1,17 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class TestDigest(models.TransientModel):
+    _name = 'digest.test'
+    _description = 'Sample Digest Wizard'
+
+    digest_id = fields.Many2one('digest.digest', string='Digest', required=True, ondelete='cascade')
+    user_ids = fields.Many2many('res.users', string='Recipients', domain="[('share', '=', False)]")
```

Comment:
We could probably add a default being "myself", as I bet this is what you are going to test generally.